sov-modules/sov-modules-api/Cargo.toml

```diff
@@ -16,6 +16,9 @@ sha2 = { workspace = true }
 bech32 = { workspace = true }
 derive_more = { workspace = true }
 
+ed25519-dalek = { version = "1.0.1" }
+rand = "0.7"
```

Comment:
We should only need randomness for key generation, right? If so, let's feature gate that behind the "native" flag.

If we need randomness for signature *verification*, then we'll probably need to use a different signature scheme - since the host can provide malicious "randomness"

Reply:
We decided on daily that the signature scheme will be implemented in the following PR.